battle_tanks/client/battle_tanks.cpp

```diff
@@ -0,0 +1,421 @@
+#include <iostream>
+#include <chrono>
+#include <memory>
+
+#include "SFML/System/Clock.hpp"
+#include "SFML/Graphics.hpp"
+#include <SFML/Network.hpp>
+#include <box2d/box2d.h>
+
+#include "utils/uuid.hpp"
+#include "game/game_objects/player_game_object.hpp"
+#include "game/sfml_game.hpp"
+#include "game/game_objects/bullet_game_object.hpp"
+#include "physics/collision_listener.hpp"
+#include "network/client_connection.hpp"
+#include "network/connection_service.hpp"
+#include "network/player_actions.hpp"
+#include "game/game_objects/forest_map_game_object.hpp"
+#include "game/game_objects/game_object_frame_restorer_packet.hpp"
+
+enum class player_game_state : unsigned int
```

Comment:
That specifically states about the network state, should be depicted in its name as well.

Reply:
done